content/en/developers/dogstatsd/data_aggregation.md

```diff
@@ -0,0 +1,59 @@
+
+This article describes why and how the aggregation is performed over your data.
+
+## Why aggregate metrics?
+
+Aggregation improves performance by reducing the number of API calls, each of which takes a certain amount of time.
+
+Consider a [COUNT metric][3] that is incremented 1,000 times (+1 each time) over a short amount of time. Instead of making 1,000 separate API calls, the DogStatsD server aggregates it into a few API calls. Depending on the situation (see below), the library may submit—for instance—1 datapoint with value 1,000 or X aggregate datapoints with a cumulated value of 1,000.
+
+## How is aggregation performed with the DogStatsD server?
+
+[DogStatsD][2] uses a *flush interval* of 10 seconds. Every 10 seconds, [DogStatsD][2] checks all data received since the last flush (i.e., in the last 10 seconds). All values that correspond to the same metric name and the same tags are aggregated together into a single value.
+
+**Note**: With the StatsD protocol, the StatsD client doesn't send metrics with timestamps. The timestamp is added at the flush time. So for a flush occurring at 10:00:10, all data received by the [DogStatsD][2] server (embedded in the Datadog Agent) between 10:00:00 and 10:00:10 is rolled up in a single datapoint that gets 10:00:00 as timestamp.
+
+## Aggregation rules per metric type
+
+Among all values received during the same flush interval, the aggregated value send depends of the [metric type][4]:
+
+| Metric Type    | Aggregation performed over one flush interval                                                                                                                  |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [GAUGE][5]     | The latest datapoint received is sent.                                                                                                                         |
+| [COUNT][3]     | The sum of all received datapoints is sent.                                                                                                                     |
+| [HISTOGRAM][6] | The min, max, sum, avg, 95 percentiles, count, and median of all datapoints received is sent. See the [HISTOGRAM metric documentation page][7] for more details. |
+| [SET][8]       | The number of different datapoints is sent.                                                                                                                     |
+| [RATE][9]      | The value difference divided by the time difference of the last 2 datapoints received.                                                                         |
```

Comment:
rate doesn't exist for dogstatsd metrics, so should be removed from here.

Also, there's one more metric type that we could mention here: `distribution`s, aggregated as [global distributions](https://docs.datadoghq.com/graphing/metrics/distributions/#overview)